lalrpop/src/lexer/nfa/mod.rs

```diff
@@ -360,7 +360,15 @@ impl NFA {
                     self.push_edge(s0, Noop, target);
                 }
                 Ok(s0)
-            }
+            },
+
+            // FIXME: Figure out what these should do
+            Expr::AnyByte => panic!("Unsupported"),
+            Expr::AnyByteNoNL => panic!("Unsupported"),
```

Comment:
These can be merged with the `AnyChar` cases -- regex supports both unicode/non-unicode, but we only use the unicode cases right now; if we ever did support bytes, it would still be doing the right thing to merge them, I think